### PR TITLE
Dpo/service portal edit screens

### PR DESCRIPTION
**Pull request by ssh16**
# Forms to edit Document provider settings 

Forms to edit document provider settings inside the service portal.

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/service-portal/document-provider/src/components/Forms/EndpointsForm.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+              placeholder={formatMessage(m.SettingsEditEndPointsUrl)}
+              value={value}
+              hasError={errors.endpoint}
+              errorMessage={errors.endpoint?.message}
+              onChange={onChange}
+            />
+          )}
+        />
+      </Stack>
+      {(renderBackButton || renderSubmitButton) && (
```
Comment (hinriksteinar) [inline libs/service-portal/document-provider/src/components/Forms/EndpointsForm.tsx:59]: Þyrftum svo sem ekki þessa lógík þar sem submit og back button ættu alltaf að vera sýnilegir. En allt í góðu svona :)

Comment (hinriksteinar) [reply libs/service-portal/document-provider/src/components/Forms/EndpointsForm.tsx:59]: Þetta á við á öllum stöðum þar sem þetta er.

Comment (hinriksteinar) [reply libs/service-portal/document-provider/src/components/Forms/EndpointsForm.tsx:59]: Þarf ekki að breyta en má alveg skoða.

**Review by hinriksteinar — APPROVED** [review]
Looks good, one minor comment. But not neccessary to change.

File: libs/service-portal/document-provider/src/components/Forms/EndpointsForm.tsx (hunk 1 above)
Comment (ssh16) [reply libs/service-portal/document-provider/src/components/Forms/EndpointsForm.tsx]: Satt, óþarfi að hafa þetta flóknara en þetta þarf að vera. Lagaði það :)

**Review by hinriksteinar — APPROVED** [review]
Looks good

**Pull request merged**